app/src/main/java/com/example/pix/chat/fragments/MusicRoomFragment.java

```diff
@@ -33,18 +49,20 @@
     // we open.
     public static Timer listenerTimer;
     private ImageView ivPlay;
+    public static Subscription<PlayerState> subscription;
```

Comment:
I want global access to our subscription with a Streamer's Spotify so that I can cancel it below.

Reply:
Not sure why you need to make it public? Do you not only use it in this class? Same question with Timer.